docs/x509-auth.md

```diff
@@ -0,0 +1,122 @@
+# Enable X.509 Authentication
+
+You can use Helm or `kubectl` to enable X.509 authentication for the 
+MongoDB Agent and client.
+
+## Prerequisites
+
+1. Add the `cert-manager` repository to your `helm` repository list and
+   ensure it's up to date:
+
+   ```
+   helm repo add jetstack https://charts.jetstack.io
+   helm repo update
+   ```
+
+1. Install `cert-manager`:
+
+   ```
+   helm install cert-manager jetstack/cert-manager --namespace cert-manager \ 
+   --create-namespace --set installCRDs=true
+   ```
+
+## Use Helm to Enable X.509 Authentication
+
+You can use Helm to install and deploy the MongoDB Community Kubernetes 
+Operator with X.509 Authentication enabled for the MongoDB Agent and 
+client. To learn more, see [Install the Operator using Helm](https://github.com/mongodb/mongodb-kubernetes-operator/blob/master/docs/install-upgrade.md#install-the-operator-using-helm).
+
+1. To deploy the MongoDB Community Kubernetes Operator, copy and paste 
```

Comment:
Since this command does a bit more than install the operator, I think it's worth mentioning here that this will deploy a sample resource with X509 enabled for both client and agent authentication and that it will also create a sample X509 user and the certificate the user can use to authenticate.